dags.yaml

```
@@ -766,3 +766,22 @@ bqetl_fog_decision_support:
   tags:
     - impact/tier_3
     - repo/bigquery-etl
+
+bqetl_newtab:
+  default_args:
+    depends_on_past: false
+    email:
+      - telemetry-alerts@mozilla.com
+      - anicholson@mozilla.com
+    email_on_failure: true
+    email_on_retry: true
+    end_date: null
+    owner: anicholson@mozilla.com
+    retries: 2
+    retry_delay: 30m
+    start_date: '2022-07-01'
```

```suggestion
    start_date: "2022-07-01"
```

^^ just for consistency

---

Seems the other `start_date`'s use single quotes. It's probably in the generation template